src/helpers/sceneless-project.ts

```diff
@@ -2012,12 +2079,33 @@ export const commands = (_project: ScenelessProject) => {
         },
       })
     },
+    setParticipantAudioOnly(
```

Comment:
Currently, calling `setParticipantAudioOnly` on ScreenShare hides the container and also mutes the audio. Making it visible again does not unmute the audio; it remains muted.

This behavior is consistent across Camera participants as well.

Reply:
It works fine for me on stage

Reply:
I think this was glitch. It works well now.